src/components/spinLoader/spinLoader.module.scss

```diff
@@ -0,0 +1,39 @@
+.spin-loader {
+  width: 16px;
+  height: 16px;
+  margin: 3px;
+  flex-shrink: 0;
+}
+
+.spinner {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  width: 100%;
+  height: 100%;
+  border-radius: 50%;
+  flex-shrink: 0;
+  position: relative;
+
+  &.color-topaz {
+    background: conic-gradient(#0000 0%, var(--rp-ui-base-topaz));
+  }
```

Comment:
Can we define it as `color-primary` and use theme-specific color here?
Just to make sure it will look nice for different themes.